autopush/main.py

```diff
@@ -97,8 +97,6 @@ def add_shared_args(parser):
     parser.add_argument('--router_write_throughput',
                         help="DynamoDB router write throughput",
                         type=int, default=5, env_var="ROUTER_WRITE_THROUGHPUT")
-    parser.add_argument('--log_level', type=int, default=40,
-                        env_var="LOG_LEVEL")
```

Comment:
Honestly, these were deprecated because they weren't used by anything, so no. Nor has there been for quite some time.